config/ingestCalibs.py

```diff
@@ -1,6 +1,6 @@
 from lsst.obs.decam.ingestCalibs import DecamCalibsParseTask
 config.parse.retarget(DecamCalibsParseTask)
-config.parse.hdu = 1
+config.parse.hdu = 0
```

Comment:
OK, good points; I will verify deleting the line has the desired effect and make an independent ticket.